Dockerfile

```diff
@@ -1,13 +1,16 @@
 FROM ubuntu:20.04
 LABEL maintainer="rubenperezm"
 
+ARG BRANCH
+
 WORKDIR /app
```

Comment:
No need to put everything within an extra subdirectory. Working at $HOME is more than enough

Reply:
Ok, changed